hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeTest.java

```diff
@@ -242,5 +251,80 @@ public void testMapCountWithAtleastOneAttribute() {
 
     }
 
+    @Test
+    public void testSetCount() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("sect"), "0");
+
+        Set<String> set1 = node.hazelcastInstance.getSet("hazelcast");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("sect"), "1");
+
+        Set<String> set2 = node.hazelcastInstance.getSet("phonehome");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("sect"), "2");
+    }
+
+    @Test
+    public void testQueueCount() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("quct"), "0");
+
+        Queue<String> queue1 = node.hazelcastInstance.getQueue("hazelcast");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("quct"), "1");
+
+        Queue<String> queue2 = node.hazelcastInstance.getQueue("phonehome");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("quct"), "2");
+    }
+
+    @Test
+    public void testMultimapCount() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mmct"), "0");
+
+        MultiMap<Object, Object> multimap1 = node.hazelcastInstance.getMultiMap("hazelcast");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mmct"), "1");
+
+        MultiMap<Object, Object> multimap2 = node.hazelcastInstance.getMultiMap("phonehome");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mmct"), "2");
+    }
+
+    @Test
+    public void testListCount() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("lict"), "0");
+
+        List<Object> list1 = node.hazelcastInstance.getList("hazelcast");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("lict"), "1");
+
+        List<Object> list2 = node.hazelcastInstance.getList("phonehome");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("lict"), "2");
+    }
+
+    @Test
+    public void testRingBufferCount() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("rbct"), "0");
+
+        Ringbuffer<Object> ringbuffer1 = node.hazelcastInstance.getRingbuffer("hazelcast");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("rbct"), "1");
+
+        Ringbuffer<Object> ringbuffer2 = node.hazelcastInstance.getRingbuffer("phonehome");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("rbct"), "2");
```

Comment:
Just for curiosity, can you please check (at least some of the tests) what happens after a datastructure gets destroyed (with its `destroy()` method)? I'm curious if the count gets decreased (as expected)